README.md

```diff
@@ -123,6 +123,22 @@ Console Class Declaration
 ["1.2.3.4"]
 ```
 
+The RSAN::Exporter class allows for the NFS mounts to be optionally available, to disable existing mounts, or prevent the mounts from installing in the first place set the following parameter:
+
+
+In Hiera
+
+```
+rsan::exporter::nfsmount:
+  - false
```

Comment:
This isn't a YAML boolean, it's an array with one value, which is a boolean. A boolean in YAML would be:

```suggestion
rsan::exporter::nfsmount: false
```

As it is now I'm pretty sure this will fail parameter validation